retriever/lib/engine.py

```diff
@@ -489,25 +491,72 @@ def download_file(self, url, filename):
                     stream=True,
                     headers={
                         'user-agent':
-                            'Weecology/Data-Retriever \
-                                            Package Manager: http://www.data-retriever.org/'
+                            'Weecology/Data-Retriever Package Manager: http://www.data-retriever.org/'
```

Comment:
This block is just the same as the one in download_file  line  L534, find away of refactoring them.